docs/troubleshooting-faq/faq.md

```diff
@@ -0,0 +1,61 @@
+# Frequently asked questions
+
+### Why didn't the sample/project I specified when running the [data download script](../getting-started/accessing-resources/getting-access-to-data.md#using-the-download-data-script) download?
+
+First, we recommend using the `--dryrun` flag when running the script to check which files _would_ be downloaded, to confirm that there is nothing wrong with your internet connection and that you are properly [logged into your AWS profile](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
```

Comment:
```suggestion
First, we recommend using the `--dryrun` flag when running the script to check which files _would_ be downloaded. 
This will confirm that there is nothing wrong with your internet connection and that you are properly [logged into your AWS profile](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
```